pulsar-broker/src/main/java/org/apache/pulsar/broker/admin/AdminResource.java

```diff
@@ -577,11 +577,9 @@ protected void internalCreatePartitionedTopic(AsyncResponse asyncResponse, int n
             Policies policies = getNamespacePolicies(namespaceName);
             maxTopicsPerNamespace = policies.max_topics_per_namespace;
         } catch (RestException e) {
-            if (e.getResponse().getStatus() != Status.NOT_FOUND.getStatusCode()) {
-                log.error("[{}] Failed to create partitioned topic {}", clientAppId(), namespaceName, e);
-                resumeAsyncResponseExceptionally(asyncResponse, e);
-                return;
-            }
+            log.error("[{}] Failed to create partitioned topic {}", clientAppId(), namespaceName, e);
```

Comment:
I'm not sure if this logic change will affect the previous logic? Can you help to confirm?

Reply:
I believe we fixed some similar problem in v2 API.
with this change if you did not create the namespace you cannot create a topic, and this makes much sense to me.

probably before this change you were able to create a topic to a non existing namespace.

we must ensure that the behaviour is consistent across v2 and v1

Reply:
Thanks for your clarification.

Reply:
> I believe we fixed some similar problem in v2 API. with this change if you did not create the namespace you cannot create a topic, and this makes much sense to me.
> 
> probably before this change you were able to create a topic to a non existing namespace.
> 
> we must ensure that the behaviour is consistent across v2 and v1

I think after this fix, we have made it's consistent.

Reply:
I think this is technically different than the v1 vs v2 problem. In this case, we're failing topic creation if getting the namespace policies fails. I think this proposed behavior is correct, since we wouldn't want to exceed the `maxTopicsPerNamespace` just because getting the namespace policies failed.

Reply:
Upon re-review, my comment is not correct. The previous code was only skipping NOT_FOUND status codes. All other errors are propagated correctly, so the only case that a topic is created when it "shouldn't be" is when the namespace policies are not found.

Reply:
@Technoboy- in my opinion, once we deprecated the V1 api, I think we should remove this logic.

Reply:
ok